modules/ROOT/pages/to-request-access.adoc

```diff
@@ -22,8 +22,8 @@ applications.
 API providers can view, manage, and revoke these contracts by following the steps in
 xref:api-manager::manage-client-apps-latest-task.adoc[Managing API Instance Contracts on API Manager].
 
-A client application's owners have permission to request access for that client application.
-To request access for a client application created by another user, ask the client application owner
+Both API instance owners and the owners of a specific client application can request access to that client application. All other users must request access from either of these owners.
+For access to a client application created by another user, request access from either the client application owner or the API instance owner.
```

Comment:
Remove period because sentence continues...probably my mistake. 
```suggestion
For access to a client application created by another user, request access from either the client application owner or the API instance owner
```